pyproject.toml

```diff
@@ -29,8 +29,68 @@ classifiers = [
     "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
 ]
-# NOTE(hadim): to fill for the pypi release
-dependencies = []
+
+dependencies = [
+    "tqdm",
+    "loguru",
+    "joblib",
+    "filelock",
+    "fsspec>=2021.9",
+    "s3fs >=2021.9",
+    "gcsfs >=2021.9",
+    "pandas",
+    "numpy",
+    "scipy",
+    "torch",
+    "datamol >=0.8.0",
+    "pyyaml",
+    "platformdirs",
+    "matplotlib",
+    "h5py",
+    "pyarrow",
+    "pydantic",
+    "platformdirs",
+    "scikit-learn",
+    "packaging",
+    "pmapper",
+    "mordred",
```

Comment:
Have you tested the pip package can install all the deps?

Reply:
yes, `pip install -e .[all]`